src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
+{
+    close();
+    app().config().features.insert(feature);
```

Comment:
I'll suggest that your implementation exclude the calls to `close()`.  That would address @mellery451's concern about unintentionally advancing the ledger.  It also would more closely model the way that an amendment goes live at a ledger boundary.

If you make this change you'll also want to include a comment in the appropriate header files that `Env::close()` must be called for the change to take effect.